targets/PythonSdk/templates/Errors.py.ejs

```diff
@@ -0,0 +1,52 @@
+from enum import Enum
+
+# Error codes returned by PlayFabAPIs
+class PlayFabErrorCode(Enum):
+    Success = 0,
+    Unknown = 1,
+    ConnectionError = 2,
+    JsonParseError = 3,
+    <% for(var i=1; i<errorList.length-1; i++) { var errorProps = errors[errorList[i]] %><%- errorProps.name %> = <%- errorProps.id %>,
+    <% } %><% var errorProps = errors[errorList[errorList.length-1]] %><%- errorProps.name %> = <%- errorProps.id %>
+
+class PlayFabError:
+
+    # the first args item is expected to be a valid dictionary
+    def __init__(self, *args):
+        if len(args) == 1:
+            self.fromJson(args[0])
+        else:
+            self.HttpCode = 0
```

Comment:
Let's default to 408

Reply:
The parameters we use here don't match the parameters we re-defined in PlayFabHttp.py